src/types.ts

```diff
@@ -1511,6 +1511,44 @@ export interface IssueTokenRequest {
     vaultAccountId: string;
     createParams: CreateTokenParams;
 }
+
+export interface JobCreatedResponse {
+    jobId: string;
+}
+
+export enum Status {
+    NONE = "NONE",
+    CREATED = "CREATED",
+    INPROGRESS = "INPROGRESS",
+    DONE = "DONE",
+    ERROR = "ERROR",
+    CANCELED = "CANCELED",
+    PAUSED = "PAUSED"
+}
+
+export class Job {
```

Comment:
This also is too general, I can think of more than one type of job. Same for `Task`, be more specific